packages/esm-commons-lib/src/api.resource.ts

```diff
@@ -145,7 +137,7 @@ export async function getPatientListsForPatient(patientUuid: string) {
 
 export function fetchPatientsFinalHIVStatus(patientUUID: string) {
   return openmrsFetch(
-    `/ws/fhir2/R4/Observation?code=${finalHIVCodeConcept}&value-concept=${finalPositiveHIVValueConcept}&patient=${patientUUID}&_sort=-date&_count=1`,
+    `/ws/fhir2/R4/Observation?code=${configSchema.obsConcepts._default.finalHIVCodeConcept}&value-concept=${configSchema.obsConcepts._default.finalPositiveHIVValueConcept}&patient=${patientUUID}&_sort=-date&_count=1`,
```

Comment:
For some reason, if i `useConfig` none of the endpoints load

Reply:
> For some reason, if i `useConfig` none of the endpoints load, i guess its because im not using a hook as the endpoint is called in a `Promise.all`